example/integrations/mpi/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
```

Comment:
can we update this to 22.04?

Reply:
I was thinking of that too, but I only tested 18.04 so far. I can test 22.04 tomorrow.

Reply:
I test 22.04 on arm64. It is OK.

Reply:
I tested 22.04, and confirmed this works on x86_64 IPv6 too.

```
$ k -n gengwg logs  lm-mpi-job-mpimaster-0
Hello world from processor lm-mpi-job-mpimaster-0, rank 0 out of 2 processors
Hello world from processor lm-mpi-job-mpimaster-0, rank 1 out of 2 processors

$ k -n gengwg get po  -w | grep mpi
lm-mpi-job-mpimaster-0         1/1     Running            0                 4m51s
lm-mpi-job-mpiworker-0         1/1     Running            0                 4m51s
lm-mpi-job-mpiworker-1         1/1     Running            0                 4m51s

$ k -n gengwg exec -it lm-mpi-job-mpimaster-0 -- mpirun --version
mpirun (Open MPI) 4.1.2
```